### PR TITLE
Rebase kolibri 016

### PR DESCRIPTION
**Pull request by manuq**
https://phabricator.endlessm.com/T33793

### CODE REVIEW

File: kolibri_explore_plugin/views.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def _get_file(self, app, path):
 
 
 class AppView(AppBase):
-    @metadata_cache
+    @method_decorator(metadata_cache, name="dispatch")
```
Comment (dbnicholson) [inline kolibri_explore_plugin/views.py:80]: I think you should squash this with the original `cache_forever` change.

**Review by dbnicholson — CHANGES_REQUESTED** [review]
This seems OK to me, but I didn't really look at the Kolibri API changes in detail. The plan is to release this as 3.x, right?

File: kolibri_explore_plugin/views.py (hunk 1 above)
Comment (manuq) [reply kolibri_explore_plugin/views.py:80]: Good point!

**Comment by manuq** [comment]
> 

Yes, this will go to v3. Merging.

**Pull request merged**